File: springboot2.x-learning/springboot-data-jpa/src/main/java/com/rtxtitanv/repository/OrderRepository.java
package com.rtxtitanv.repository;

import com.rtxtitanv.model.Order;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;

/**
 * @author rtxtitanv
 * @version v1.0.0
 * @name com.rtxtitanv.repository.OrderRepository
 * @description OrderRepository接口用于操作orders表
 * @date 2020/1/3 21:42
 */
public interface OrderRepository extends JpaRepository<Order, Long>, JpaSpecificationExecutor<Order> {
}

File: springboot2.x-learning/springboot-data-jpa-kotlin/src/main/kotlin/com/rtxtitanv/repository/OrderRepository.kt
package com.rtxtitanv.repository

import com.rtxtitanv.model.Order
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor

/**
 * @name com.rtxtitanv.repository.OrderRepository
 * @description OrderRepository接口用于操作orders表
 * @author rtxtitanv
 * @date 2020/1/20 22:32
 * @version 1.0.0
 */
interface OrderRepository : JpaRepository<Order, Long>, JpaSpecificationExecutor<Order>